File: python/concatenateArray.py
https://leetcode.com/problems/concatenation-of-array/
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:

        ans = []

        #concatenation of two nums arrays, x=2
        for i in range(2):
            for n in nums:
                ans.append(n)
        return ans

File: Go/concatenateArray.go
func getConcatenation(nums []int) []int {
    var ans []int

    for i := 0; i < 2; i++ {
        for _, n := range nums { // Use underscore (_) to ignore the index
            ans = append(ans, n) // Append the element n to ans
        }
    }
    return ans
}

